pymc/distributions/multivariate.py

```diff
@@ -2452,6 +2453,15 @@ class ICAR(Continuous):
     rv_op = icar
 
     @classmethod
+    def ICAR(name, W, tau=1.0, mu=None, **kwargs):
```

Comment:
Doesn't seem like you want the classmethod decorator here. It should stay on `dist` method .